controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -169,6 +165,13 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 		// GetDaemonResources()
 		Resources: &endpointResources,
 	}
+	// Add autoscale type for noobaa CR
+	if nb.Spec.Autoscaler.AutoscalerType == "" {
```

Comment:
Do you want it to change if and only if it is empty? what if it is `keda`? Earlier it looks like we always wanted to have it to `hpav2`.

Reply:
Hi @iamniting It can never be `Keda` unless the user updates it manually. Since we are hardcoding `hpav2` AutoscalerType always will have that value. Previous code also not restricting users from updating the AutoscalerType with the value keda. Correct me if am wrong.

Reply:
We have several reconciliation strategies, and one of them involves ignoring certain changes made by the user. In this case, users can update the values manually, and we won't revert them back to the original state. However, if the user decides to switch back to a managed state, we will reconcile the values to their original state. Additionally, if the user changed a value to 'keda' while it was in the ignored state, we are currently not reverting it back, but in my opinion, we should do so.

Can you pls check with Danny or someone if we ever supported changing this value or can it be `keda` in any case?

Reply:
@iamniting, With this release, our plan is to go only with HPAV2 for our side. So I think it's better to avoid the keda in any case. I will update the code.